draft-ietf-quic-transport.md

```diff
@@ -2499,9 +2499,11 @@ A connection that remains idle for longer than the advertised idle timeout (see
 draining state when the idle timeout expires.
 
 Each endpoint advertises their own idle timeout to their peer. The idle timeout
-starts from the last packet received, or the first packet sent after the last
-received packet.  The latter condition ensures that initiating new activity
-postpones a timeout.
+starts from the last packet received.  In order to ensure that initiating new
+activity postpones an idle timeout, an endpoint restarts this timer when sending
+a packet if there are no other packets outstanding.  This condition only applies
```

Comment:
How about "In order to ensure that initiating new activity postpones an idle timeout, an endpoint restarts this timer when sending a packet containing frames besides ACK or PADDING and no other packets are outstanding."